modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/UserCard.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayCardWithUser} from '@clayui/card';
+import React, {useState} from 'react';
+
+const getDataAttributes = (data) => {
```

Comment:
Since we're going to be duplicating this function: both in this PR and in mine we should probably plan a future task to put it in it's own file to be able to re-use it. I'm also wondering why you're using a function declaration instead of a function expression.

Reply:
> Since we're going to be duplicating this function: both in this PR and in mine we should probably plan a future task to put it in it's own file to be able to re-use it

Sure, we can extract it in the next pr.

> I'm also wondering why you're using a function declaration instead of a function expression.
Just a matter of habit, I think there's no difference in this case.

Reply:
:+1: and :+1: